ethereum/spec/src/test/resources/tech/pegasys/teku/spec/config/standard/presets/mainnet/merge.yaml

```diff
@@ -18,4 +18,13 @@ MAX_TRANSACTIONS_PER_PAYLOAD: 1048576
 # 2**8 (= 256)
 BYTES_PER_LOGS_BLOOM: 256
 # 2**5 (= 32)
-MAX_EXTRA_DATA_BYTES: 32
\ No newline at end of file
+MAX_EXTRA_DATA_BYTES: 32
+
+# Transition
+# ---------------------------------------------------------------
+# TBD, 2**256-2**10 is a placeholder
+TERMINAL_TOTAL_DIFFICULTY: 115792089237316195423570985008687907853269984665640564039457584007913129638912
+# 0
+TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
+# FAR_FUTURE_EPOCH
+TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
```

Comment:
These don't belong in the presets file.  They should only exist in the network definition file (files with the `PRESET_BASE` key).  They're the files in `util/src/main/resources/tech/pegasys/teku/util/config/configs`

So none of the config files changed currently should have it. I think we should then be removing `TRANSITION_TOTAL_DIFFICULTY` since it's now been renamed.

Importantly, we need to check that we can still create a Spec by parsing the output of the config endpoint from before this change.  ie should be able to use a `--network=auto` on a validator client with this change with a beacon node without it.  Otherwise we're breaking backwards compatibility and it's going to cause issues.